docs/howto/custom_json_encoder_decoder.rst

```diff
@@ -0,0 +1,49 @@
+Define custom JSON encoders and decoders
+----------------------------------------
+
+When calling ``mytask.defer()`` to create a job, the task arguments are serialized into
+a JSON string for storing into the Postgres database.
+
+And after fetching a job from the database a Procrastinate worker needs to deserialize
+the task arguments before calling the task.
+
+By default Procrastinate relies on the JSON dumps and loads functions used by psycopg2.
+See the `psycopg2 documentation`_. But Procrastinate makes it possible to specify custom
```

Comment:
`See the `psycopg2 documentation` ` entre parenthèses, et ne pas couper la phrase en 2 ou enlever "But"

Reply:
Done.